lib/webview-helpers.js

```diff
@@ -370,6 +368,31 @@ helpers.getWebviews = async function getWebviews (adb, {
   return result;
 };
 
+/**
+ * Retrieves web views mapping for getContexts
+ *
+ * @param {object} adb - an ADB instance
+ * @param {GetWebviewOpts} opts (see note on getWebviews).
+ *
+ * @return {Array<WebviewProps>} webviewsMapping(see note no collectWebviewsDetails).
+ */
+helpers.getWebViewsMapping = async function getWebViewsMapping (adb, {
```

Comment:
The names of web views returned by this method might differ from the ones returned in `getWebviews`. Please unify them

Reply:
Thank you for your comment. I think current implementation meets your comment. Now the object in returned value of  `getWebViewsMapping` will have a property named `webviewName` (e.g.  `WEBVIEW_<package_name>`). 

However, the value `webviewName` will be `undefined` when it reaches to https://github.com/appium/appium-android-driver/blob/master/lib/webview-helpers.js#L357 

Should `webviewName` have any dummy value other than `undefined`?

Reply:
> Should webviewName have any dummy value other than undefined?

You could set the name to `null` for this case

Reply:
Thank you. It's done.